sdk/core/Microsoft.Azure.Core.Spatial.NewtonsoftJson/tests/Samples/Readme.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure;
+using Azure.Core.Serialization;
+using Azure.Core.TestFramework;
+using Azure.Search.Documents;
+using Azure.Search.Documents.Indexes;
+using Azure.Search.Documents.Indexes.Models;
+using Azure.Search.Documents.Models;
+using Microsoft.Spatial;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using NUnit.Framework;
+
+namespace Microsoft.Azure.Core.Spatial.NewtonsoftJson.Tests.Samples
+{
+    public class Readme
+    {
+        [Test]
+        public void SearchSample()
+        {
+            MockResponse response = new MockResponse(200);
+            response.SetContent(@"{
+                ""value"": [
+                    {
+                        ""@search.score"": 1.0,
+                        ""id"": ""1"",
+                        ""name"": ""Mount Rainier"",
+                        ""summit"": {
+                            ""type"": ""Point"",
+                            ""coordinates"": [
+                                -121.76044,
+                                46.85287
+                            ]
+                        }
+                    }
+                ]
+            }");
+
+            Environment.SetEnvironmentVariable("SEARCH_ENDPOINT", "https://sample.search.windows.net");
+            Environment.SetEnvironmentVariable("SEARCH_API_KEY", "sample");
+
+            Mountain _mountain = null;
+
+            #region Snippet:Microsoft_Azure_Core_Spatial_NewtonsoftJson_Samples_Readme_SearchSample
+            // Get the Azure Cognitive Search endpoint and read-only API key.
+            Uri endpoint = new Uri(Environment.GetEnvironmentVariable("SEARCH_ENDPOINT"));
+            AzureKeyCredential credential = new AzureKeyCredential(Environment.GetEnvironmentVariable("SEARCH_API_KEY"));
+
+            // Create serializer options with our converter to deserialize geographic points.
+            JsonSerializerSettings serializerSettings = new JsonSerializerSettings
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                Converters =
+                {
+                    new NewtonsoftJsonMicrosoftSpatialGeoJsonConverter()
+                }
+            };
+
+            SearchClientOptions clientOptions = new SearchClientOptions
+            {
+                /*@@*/ Transport = new MockTransport(response),
+                Serializer = new NewtonsoftJsonObjectSerializer(serializerSettings)
+            };
+
+            SearchClient client = new SearchClient(endpoint, "mountains", credential, clientOptions);
+            Response<SearchResults<Mountain>> results = client.Search<Mountain>("Rainier");
+
+            foreach (SearchResult<Mountain> result in results.Value.GetResults())
+            {
+                Mountain mountain = result.Document;
+                /*@@*/ _mountain = mountain;
```

Comment:
I think you can just re-enumerate the results.

Reply:
I thought about it, but this is cheaper. :) I can change it if you really want me to.

Reply:
I don't like the snippet processor directives, that's all :)